share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -332,9 +332,9 @@ Ensure you reference any relevant GitHub issue e.g. `Closes #12345` in the commi
 
 Now you just need to push your commit to GitHub.
 
-If you haven’t forked Homebrew yet, [go to the repo and hit the fork button](https://github.com/Homebrew/homebrew).
+If you haven’t forked Homebrew yet, [go to the homebrew-core repo and hit the fork button](https://github.com/Homebrew/homebrew-core).
 
-If you have already forked Homebrew on GitHub, then you can manually push (just make sure you have been pulling from the Homebrew/homebrew master):
+If you have already forked Homebrew on GitHub, then you can manually push (just make sure you have been pulling from the Homebrew/homebrew-core master):
 
 ```shell
 git push https://github.com/myname/homebrew/ <what-you-called-your-branch>
```

Comment:
`https://github.com/myname/homebrew/` here should also be updated to `https://github.com/myname/homebrew-core/`.